app/services/return-versions/setup/check/generate-return-version.service.js

```diff
@@ -49,17 +50,23 @@ async function _generateReturnRequirements (sessionData) {
 async function _generateReturnVersion (nextVersionNumber, sessionData, userId) {
   const startDate = new Date(sessionData.returnVersionStartDate)
   let endDate = null
+  let quarterlyReturns = false
 
   if (nextVersionNumber > 1) {
     endDate = await ProcessExistingReturnVersionsService.go(sessionData.licence.id, startDate)
   }
 
+  if (isQuarterlyReturnSubmissions(sessionData.returnVersionStartDate)) {
+    quarterlyReturns = sessionData.quarterlyReturns
+  }
+
   return {
     createdBy: userId,
     endDate,
     licenceId: sessionData.licence.id,
     multipleUpload: sessionData.multipleUpload,
     notes: sessionData?.note?.content,
+    quarterlyReturns,
```

Comment:
I do think this would be better as a ternary but i am not sure we do this elsewhere. 

Reason for the false is that quarterlyReturns can be undefined when it has not been set. 

Unlike multiple-upload which is set when the session is instantiated.

```js
  quarterlyReturns: isQuarterlyReturnSubmissions(sessionData.returnVersionStartDate) ? sessionData.quarterlyReturns : false
```

Reply:
I think in this case a ternary is fine, and I have no preference either way so happy if you want to switch to use it.